packages/zoe/src/contracts/gimix/start-postalSvc.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @file core eval script* to start the postalSvc contract.
+ *
+ * * see test-gimix-proposal.js to make a script from this file.
+ *
+ * The `permit` export specifies the corresponding permit.
+ */
+// @ts-check
+
+import { E, Far } from '@endo/far';
+
+const trace = (...args) => console.log('start-postalSvc', ...args);
+
+const fail = msg => {
+  throw Error(msg);
+};
+
+/**
+ * ref https://github.com/Agoric/agoric-sdk/issues/8408#issuecomment-1741445458
+ *
+ * @param {ERef<import('@agoric/vats').NameAdmin>} namesByAddressAdmin
+ */
+const fixHub = async namesByAddressAdmin => {
+  /** @type {import('@agoric/vats').NameHub} */
+  // @ts-expect-error mock. no has, keys, ...
```

Comment:
'Mock' is a term for testing. Is there value in calling this `NameHub` when it manifestly isn't?

Would it work to declare the type to be just `{ lookup: (...path: string[]) => Promise<any> }` to match a subset of NameHub?